app/models/user.rb

```diff
@@ -54,25 +54,62 @@ def administrated_projects
     scope
   end
 
-  def self.to_csv
-    @users = User.order(:id)
+  def self.to_csv(
```

Comment:
sounds good!

On Tue, Aug 2, 2016 at 1:51 PM, Robert Ikeoka <notifications@github.com>
wrote:

> In app/models/user.rb
> <https://github.com/zendesk/samson/pull/1132#discussion_r73234742>:
>
> > @@ -54,25 +54,62 @@ def administrated_projects
> >      scope
> >    end
> >
> > -  def self.to_csv
> > -    @users = User.order(:id)
> > +  def self.to_csv(
>
> should the csv_line and effective_project_role also be moved to the
> presenter given they aren't using the native model methods. It'll keep the
> optimized logic outside of the model, that won't matter for other use cases.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1132/files/13cf06f9fb82ef8dddf922856ec64e87df5cea81#r73234742>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZw0o-olHVBP6RncTjlcXq8dq2WOpks5qb622gaJpZM4JK04R>
> .
>